web/app/plugins/mitlib-multisearch-widget/templates/tab-more-alma.php

```diff
@@ -17,6 +17,7 @@
 			<li><a href="/theses">MIT theses</a></li>
 			<li><a href="https://dspace.mit.edu">DSpace@MIT</a></li>
 			<li><a href="https://archivesspace.mit.edu/">ArchivesSpace</a></li>
+			<li><a href="https://dataverse.harvard.edu/dataverse/mit-libraries">MIT Libraries Dataverse</a></li>
 			<li><a href="http://libraries.mit.edu/experts/">Search tools by subject</a></li>
```

Comment:
I know you didn't want to touch things outside the main scope of the ticket, but can you please make this `https` or  move it to a relative link?

Reply:
I just pushed a commit to address the two `http://` protocol links in this partial - good catch, thanks!

I haven't pushed this to Pantheon, but can if you'd like.

Reply:
This change is simple enough I don't feel a need to see the multi dev. Thanks!